examples/binding_affinity_prediction/utils.py

```diff
@@ -46,20 +64,36 @@ def load_dataset(args):
     """
     assert args['dataset'] in ['PDBBind'], 'Unexpected dataset {}'.format(args['dataset'])
     if args['dataset'] == 'PDBBind':
-        dataset = PDBBind(subset=args['subset'],
+        if args['model'] == 'PotentialNet': 
+            from functools import partial
+            from dgllife.utils import potentialNet_graph_construction_featurization
+            dataset = PDBBind(subset=args['subset'], pdb_version=args['version'], 
+                    remove_coreset_from_refinedset=args['remove_coreset_from_refinedset'],
```

Comment:
Fix the indent for L71 - L75, i.e. the first character should align with the first "s" in "subset"